internal/pkg/store/sql/sqlite/database.go

```diff
@@ -1,10 +1,10 @@
-// Copyright 2022-2022 EMQ Technologies Co., Ltd.
+// Copyright 2023-2023 EMQ Technologies Co., Ltd.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+// http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Here, can we revert? Otherwise, this will cause meaningless diff if others change this file.